src/cocotb_tools/runner.py

```diff
@@ -60,6 +60,12 @@ def shlex_join(split_command):
     return " ".join(shlex.quote(arg) for arg in split_command)
 
 
+class VHDL(str): ...
+
+
+class Verilog(str): ...
+
+
```

Comment:
Something like this? Having them blank in the documentation is a bit confusing.
```suggestion
class VHDL(str): ...
    """Wrapper to tag string arguments to :meth:`~cocotb_tools.runner.Simulator.build` as VHDL-specific."""

class Verilog(str): ...
    """Wrapper to tag string arguments to :meth:`~cocotb_tools.runner.Simulator.build` as Verilog-specific."""


```